dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/ValidationUtils.java

```diff
@@ -75,7 +73,6 @@ private ValidationUtils() {
 
   public static void validateGeometry(
       Reporter reporter, TrackerDto dto, Geometry geometry, FeatureType featureType) {
```

Comment:
Add `@Nonnull` to help us remember and help static analysis. Here and where applicable.